### PR TITLE
Fix 3D Tiles Inspector Styling

### LINKED ISSUE #7028: 3D Tiles Inspector looks funny in master
3d-tiles-inspector-bug

Seems to have happened after the 1.49 release.

### PR DESCRIPTION
**Pull request by hpinkos**
Fixes #7028

No need to update `CHANGES.md` since the wrong styling wasn't released

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/tileset-inspector-fix/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by likangning93** [comment]
The onscreen area below the inspector no longer seems to detect mouse click events or work for mouseover picking, even with the inspector collapsed. Is this intended?

Also, not introduced in this PR, but sliders don't provide visual feedback in IE. They still work though, so maybe that's not a big deal:

sliders

**Comment by hpinkos** [comment]
Good catch @likangning93!  We had all this crazy range input styling.  It's better just to use the browser defaults.

This is ready for another look

File: Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js [hunk 1]
```diff
@@ -221,27 +225,19 @@ define([
         return destroyObject(this);
     };
 
-    function makeSection(name, visibleProp, toggleProp, contents) {
-        var toggle = document.createElement('span');
-        toggle.className = 'cesium-cesiumInspector-toggleSwitch';
-        toggle.setAttribute('data-bind', 'text: ' + visibleProp + ' ? "-" : "+", click: ' + toggleProp);
-
+    function makeSection(panel, name, visibleProp, toggleProp, contents) {
```
Comment (SunBlack) [inline Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js]: Synchronize implementation with `CesiumInspector::addSection`. Maybe a base class for both inspectors would be a good idea?

File: Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js [hunk 2]
```diff
@@ -52,13 +52,25 @@ define([
...
         var updatePanelContents = document.createElement('div');
+        updatePanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var loggingPanelContents = document.createElement('div');
+        loggingPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var tileDebugLabelsPanelContents = document.createElement('div');
+        tileDebugLabelsPanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var stylePanelContents = document.createElement('div');
+        stylePanelContents.className = 'cesium-cesiumInspector-sectionContent';
         var optimizationPanelContents = document.createElement('div');
+        optimizationPanelContents.className = 'cesium-cesiumInspector-sectionContent';
```
Comment (SunBlack) [inline Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js]: A lot of code duplication. You can prevent it by using `CesiumInspector::addSection` (remove `makeSection` later).

File: Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js [hunk 3]
```diff
@@ -156,22 +170,12 @@ define([
...
-        element.appendChild(updatePanel);
-        element.appendChild(loggingPanel);
-        element.appendChild(tileDebugLabelsPanel);
-        element.appendChild(stylePanel);
-        element.appendChild(optimizationPanel);
+        makeSection(panel, 'Tileset', 'tilesetVisible', 'toggleTileset', tilesetPanelContents);
+        makeSection(panel, 'Display', 'displayVisible', 'toggleDisplay', displayPanelContents);
+        makeSection(panel, 'Logging', 'loggingVisible', 'toggleLogging', loggingPanelContents);
+        makeSection(panel, 'Tile Debug Labels', 'tileDebugLabelsVisible', 'toggleTileDebugLabels', tileDebugLabelsPanelContents);
+        makeSection(panel, 'Style', 'styleVisible', 'toggleStyle', stylePanelContents);
```
Comment (likangning93) [inline Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js]: Looks like the `Update` panel is missing?

File: Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js [hunk 4]
```diff
@@ -156,22 +170,12 @@ define([
...
-        // first add and bind all the toggleable panels
-        element.appendChild(tilesetPanel);
-        element.appendChild(displayPanel);
-        element.appendChild(updatePanel);
-        element.appendChild(loggingPanel);
-        element.appendChild(tileDebugLabelsPanel);
-        element.appendChild(stylePanel);
-        element.appendChild(optimizationPanel);
+        makeSection(panel, 'Tileset', 'tilesetVisible', 'toggleTileset', tilesetPanelContents);
+        makeSection(panel, 'Display', 'displayVisible', 'toggleDisplay', displayPanelContents);
```
Comment (likangning93) [inline Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js]: The `Point Cloud Shading` options under `display` used to be hidden when the box was unchecked, is that still doable? If you need it, here's some Sandcastle code for making sure everything is still linking up correctly, but all the parameters seem to work fine as of 75c731b.

```js
var viewer = new Cesium.Viewer('cesiumContainer');
viewer.extend(Cesium.viewerCesium3DTilesInspectorMixin);
var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(5713) });
viewer.scene.primitives.add(tileset);
tileset.readyPromise.then(function(){
    viewer.zoomTo(tileset);
});
```

**Comment by hpinkos** [comment]
Thanks for the input @SunBlack @likangning93.  That first pass was kind of rushed.

I created an `InspectorShared` static class so both widgets use the same `createCheckbox` and `createSection` functions.  I also fixed the element visibility issues and added back the missing section from the 3d tiles inspector.

Anything else?

**Comment by SunBlack** [comment]
Source looks okay for me now 👍 

Just a suggestion: Can't we replace gMinusButton/gPlusButton in CesiumInspector by
`<input type="range" min="1" max=that._numberOfFrustums>`

**Comment by hpinkos** [comment]
> Can't we replace gMinusButton/gPlusButton in CesiumInspector

@SunBlack That would be fine with me.  I can't spend any more time on this right now though.  Feel free to open another PR with any more changes =)

**Comment by likangning93** [comment]
Thanks @hpinkos!

**Pull request merged**